docs/userguide.md

```diff
@@ -298,6 +298,27 @@ done
 If you were able to verify the resources removed and added back then you have
 successfully verified a working federation-v2 deployment.
 
+### Create ReplicaSchedulingPreference
+
+We can use `ReplicaSchedulingPreference` to adjust resources across different clusters.
```

Comment:
We should probably be a little more specific here - what about this?

"ReplicaSetSchedulingPreferences allow users to maintain a total number of replicas across all clusters where a Deployment or ReplicaSet is placed by federation."

cc @irfanurrehman

Reply:
Thanks @pmorie , I updated it as follows:

`ReplicaSetSchedulingPreferences` allow users to maintain a total number of replicas
across all clusters where a Deployment or ReplicaSet is placed by federation, it can
also adjust the replica across clusters based on `ClusterPreferences`.

/cc @irfanurrehman